### PR TITLE
Allow inim to be started with any -d flags

### PR DESCRIPTION
**Pull request by ryanc-signiq**
Postional arguments at the end of the inim invocation get passed to the compiler

ie inim --nim=~/.nim/mynim -d ssl -d release

### CODE REVIEW

File: src/inim.nim [hunk 1]
```diff
@@ -240,7 +241,11 @@ proc runForever() =
     while true:
         # Read line
         try:
-            currentExpression = readLineFromStdin(getPromptSymbol()).strip
+            let result = readLineFromStdin(getPromptSymbol(), currentExpression)
+            if not result:
+                raise newException(EOFError, "Ctrl+D was pressed")
+        except EOFError:
+            cleanExit()
```
Comment (0atman) [inline src/inim.nim:248]: Looks like there are some unrelated commits in this PR

File: src/inim.nim [hunk 2]
```diff
@@ -7,14 +7,15 @@ type App = ref object
...
+    flags: string
 
 var app: App
 
 const
     INimVersion = "0.4.1"
     indentSpaces = "    "
     indentTriggers = [",", "=", ":", "var", "let", "const", "type", "import",
-                      "object", "enum"] # endsWith
+                      "object", "RootObj", "enum"] # endsWith
```
Comment (0atman) [inline src/inim.nim:18]: Unrelated?

**Comment by Tangdongle** [comment]
Hey mate, I've removed the unrelated code from the commit. Apologies for the unrelated code ;)

**Comment by 0atman** [comment]
This is now in release 0.4.2 :ok_hand:
